h2o-core/src/main/java/water/H2O.java

```diff
@@ -2038,6 +2039,10 @@ public static void main( String[] args ) {
     if (GA != null)
       startGAStartupReport();
 
+    // Set parser timezone to UTC by default
+    ParseTime.setTimezone("UTC");
```

Comment:
You can avoid send the MRTask simply by setting the private static property:
```
private static DateTimeZone _timezone = DateTimeZone.forID("URC")
```

Reply:
The MRTask.doAllNodes() is necessary to make the update on all the nodes in the cluster, not just the master node.

Reply:
It is not necessary :) All nodes have same copy of the code, all nodes will be statically initialized ;)

Reply:
Ah, good point.  I didn't look carefully enough to see that it was a static initializer.